src/forms/_base.scss

```diff
@@ -226,6 +235,11 @@ select {
     transform: translateY(1rem) scale(1);
     transition: transform 0.25s ease-out;
     transform-origin: 0 0;
+    text-transform: none;
+
+    + .form-input {
```

Comment:
I think it's standard in the Casper codebase to use the `&` explicitly except when doing simple descendent nesting. I would be down to talk about a global refactor.